Agnes Pravida Irlitashanty

agnespravida opened the pull request:
secret key jwt = secretKey

agnespravida committed:
rest-api and documentation

agnespravida committed:
Finish crud routing

dhimashary commented:
register belum ada

dhimashary commented:
model user belum ada

agnespravida committed:
add login and register features

agnespravida committed:
authentication and authorization

agnespravida committed:
error handling

agnespravida committed:
add nodemailer

agnespravida committed:
create and read client side

agnespravida committed:
add update delete client

agnespravida committed:
update status client

dhimashary commented:
env yang digunakan comment disini ya

dhimashary commented:
nodemailernya masukin folder helpers ya

dhimashary commented:
di nodemailer 
if (error) {
      cb(err) <--- harusnya error

dhimashary commented:
karena error yg ditampilkan di looping semua, ketika dapat error string, per hurufnya ikut kena looping

dhimashary commented:
fungsionalitas udah oke., kurang di layout nya aja belum rapi

agnespravida committed:
1. Separating helpers function from index.html in client side to folder assets
2. Separating main function jquery from index html in client to folder assets

agnespravida committed:
1. Change layout of todos list from accordion to card on function showTodos in helpers file

agnespravida committed:
1. Menghapus input status pada fungsi replaceTodo pada controller todo
2. Menghapus input form status pada form add pada index.html
3. Menghapus input form status pada form edit pada index.html
4. Mengkomen/non aktifkan proses pengambilan nilai dari field status pada form edit pada fungsi helpers (line 212)

agnespravida committed:
1. Add fullCalendar library
2. Integrating todo-list and event on fullCalendar

agnespravida committed:
1. Change register and login form to card
2. Creating navbar in main page

agnespravida committed:
1. OAuth Google

agnespravida committed:
1. Add API (favqs) router
2. Add favqs controller
3. Add favqs layout

agnespravida committed:
1. Add placeholder for login and register form

agnespravida committed:
1. Add reminder feature
2. Add getDeadline function on helpers (to get show closest deadline to the user)
3. Add closest deadline calculation to the showTodo function

dhimashary commented:
nice approach banget di design kalendernya, mantap, lebih keren lagi kalo ada input jamnya bisa kapan
bisa lebih baik lagi kalo form add dan edit todonya juga rapi

dhimashary commented:
document html title nya jangan "document" ya

agnespravida committed:
1. Fix register, login, add, and edit form
2. Add confirmation to delete feature
3. Fix layout
